examples/phewas_scatter.html

```diff
@@ -116,33 +256,93 @@ <h6 style="float: right;"><a href="../index.html">&lt; return home</a></h6>
           id: "variant",
           type: "orthogonal_line",
           orientation: "vertical",
-          offset: 123352136,
+          offset: variantPosition,
           style: {
             "stroke": "#FF3333",
             "stroke-width": "2px",
             "stroke-dasharray": "4px 4px"
           }
         });
 
+        // Modify the tooltips for PheWAS result data layer points to contain more data. The fields in this sample
+        //   tooltip are specific to the LZ-Portal API, and are not guaranteed to be in other PheWAS datasources.
+        var phewas_layer = layout.panels[0].data_layers[1];
+        // Tell the layer to also fetch some special fields; otherwise the datasource will hide this info (TODO)
+        phewas_layer.fields.push("phewas:pmid", "phewas:description", "phewas:study");
+        phewas_layer.tooltip.html = [
+            "<strong>Trait:</strong> {{phewas:trait_label|htmlescape}}<br>",
+            "<strong>Trait Category:</strong> {{phewas:trait_group|htmlescape}}<br>",
+            "<strong>P-value:</strong> {{phewas:log_pvalue|logtoscinotation|htmlescape}}<br>",
+
+            "{{#if phewas:study}}",
+              "<br><strong>Study:</strong> {{phewas:study|htmlescape}}<br>",
```

Comment:
The extra line break leading this line creates extra unneeded white space in the tool tips and should probably be removed.